libsplinter/protos/admin.proto

```diff
@@ -310,8 +310,7 @@ message AdminMessage {
         CONSENSUS_MESSAGE = 1;
         PROPOSED_CIRCUIT = 2;
         MEMBER_READY = 3;
-        DISBANDED_CIRCUIT = 4;
```

Comment:
This commit seems like two very different changes.  They should probably be split up.

Reply:
Updated the change to the admin.proto file into a separate commit